x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_view.tsx

```diff
@@ -121,6 +121,7 @@ export const AnalyticsCollectionView: React.FC = () => {
           ? [
               <RedirectAppLinks application={application}>
                 <EuiLink
+                  data-test-subj="enterpriseSearchAnalyticsCollectionViewLink"
```

Comment:
Hello 👋 

We are using data-test-id and a different data shape at EnterpriseSearch. Can we configure this maybe?